src/components/base-layout.vue

```diff
@@ -3,13 +3,28 @@
     <nav class="fr-container" role="navigation" aria-label="Accès rapide">
       <ul class="fr-skiplinks__list">
         <li>
-          <router-link class="fr-link" to="#main">Contenu</router-link>
+          <router-link
+            class="fr-link"
+            :to="{ hash: '#main' }"
+            :aria-current="none"
```

Comment:
none n'est pas défini ici ou c'est une chaine de caractères ?
```suggestion
            aria-current="none"
```

Reply:
Merci pour l'éclaircissement